core/shared/src/main/scala/matryoshka/package.scala

```diff
@@ -35,101 +35,103 @@ import scalaz._, Scalaz._
   * @groupdesc dist Natural transformations required for generalized folds and unfolds.
   */
 package object matryoshka {
+  type Fix[F[_]] = matryoshka.data.Fix[F]
+  val Fix        = matryoshka.data.Fix
```

Comment:
We had like one brief day where the `compile` flow in Quasar used `Mu`, but then a change was made that made it difficult to use different fixed-points for different paths. Which is fine, but it meant the other paths being stuck on Fix made the decision of _which_ fixed point for us.